### ISSUE TITLE
NTR: Donath Landsteiner autoantibody test abnormal (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8791]
New term request Donath Landsteiner autoantibody test abnormal (segal_211224211044):   UMLS: 0272129

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000387
- label: Anti-D-L antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:34433582
- def: The presence of Donath-Landsteiner (D-L) autoantibodies (immunoglobulins) in the serum.
- comment: This feature may be observed in paroxysmal cold haemoglobinuria.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8791
- synonyms: Anti-Donath-Landsteiner antibody positivity(NONE)
